README.md

```diff
@@ -255,18 +253,20 @@ if err != nil {
 }
 ```
 
+<!-- GO EXAMPLE
+}
+-->
+
 ## Known limitations (work in progress)
 
 As the library is actively developed couple of features are not there yet:
 
-- Realtime connection recovery is not implemented
-- Realtime connection failure handling is not implemented
-- ChannelsOptions and CipherParams are not supported when creating a Channel
+- Realtime connection failure handling is partially implemented
```

Comment:
This is surprising, what is left to implement? I think this is one of the main features we're aiming to give to customers in this release.

Reply:
Once #172 is merged this line can be removed, but the API preview release that will result from _this_ PR won't have that (as things stand now).